build.sbt

```diff
@@ -25,6 +25,8 @@ val javaDslApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-
 val javaApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-java/api/"
 val javaAndroidApiUrl = s"http://fauna.github.io/faunadb-jvm/$driverVersion/faunadb-android/api/"
 
+useGpg := true
```

Comment:
So, just out of curiosity, how's this going to work on the CI?
Also, is ti possible to move it to `publishSettings`? We were trying to keep all configs related to publishing artifacts in there.